src/pudl_archiver/frictionless.py

```diff
@@ -14,6 +14,7 @@
     "zip": "application/zip",
     "xlsx": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
     "csv": "text/csv",
+    "txt": "text/csv",
```

Comment:
@katie-lamb This is the description update I was referring to. Not sure if having both csv and txt have the same description is an issue, but this is accurate.

Reply:
Hmm seems fine I think.